_partials/_cloud-mst-comparison.md

```diff
@@ -2,12 +2,11 @@
   type="cloud"
   header="Timescale or Managed Service for TimescaleDB?"
 >
-If your workload lives in AWS, use
-[Timescale](/getting-started/latest/),
-built and exclusively operated by Timescale, designed to offer maximum
-cost-effectiveness and performance for time-series data. If you need Azure or
-GCP, contact
+If your workload lives in AWS, use [Timescale](/getting-started/latest/). Built
+and exclusively operated by Timescale, Timescale is a PostgreSQL data platform
+for time-series, events and analytics.
+
+If you need Azure or GCP, contact
```

Comment:
```suggestion
If you need Azure or GCP, contact
```
Do we need this box?  Can we remove it entirely?  If we can't remove it, can we simply remove from "If you need Azure or GCP..."

Reply:
So the point of this partial is to compare the two. We use it mostly on MST pages, to try and direct people back to Cloud. For example: https://docs.timescale.com/mst/latest/

So, with that in mind, I don't think we can avoid talking about MST directly, which is why I removed the sign up link and redirected people to support. That way, if they're desperate enough for MST, we can get them to talk to someone directly, and qualify whether or not they actually need it.